src/map/handler/Map.Drag.js

```diff
@@ -96,12 +96,12 @@ export var Drag = Handler.extend({
 	},
 
 	_onDown: function () {
```

Comment:
Yeah, the solution LGTM. I don't think that the behaviour in the cases it affects (inertia drag, then the user presses the button, waits, then drags) will be noticeable.